styleguide/components/11-base/fonts/_fonts.scss

```diff
@@ -1 +1,5 @@
+////
+/// This file defines fonts.
+////
+
 @import url('https://fonts.googleapis.com/css?family=Fira+Sans'); // sass-lint:disable-line no-url-protocols
```

Comment:
Maybe only include used font-weights and variants.

Reply:
Added.